scripts/memfault/README.rst

```diff
@@ -0,0 +1,85 @@
+.. _mds_ble_gateway_script:
+
+MDS BLE gateway script
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The MDS BLE gateway is an application used as a client for the :ref:`mds_readme`.
+It forwards data received on the Memfault Diagnostic Service to the Memfault`s cloud for further analysis.
+The destination data URI and the Memfault project key are read from the service characteristics.
+
+Overview
+********
+
+The script scans for the :ref:`mds_readme` service UUID in the advertising packets.
+Once it finds the MDS service UUID, it connects to the remote device and reads the MDS data.
+The scripts waits for the MDS notifications and forwards them to the URI read from the URI characteristics.
+
+For more details about the Memfault, see `Memfault SDK`_ and :ref:`mod_memfault` for details on integration with the |NCS|.
+
+Requirements
+************
+
+The script source files are located in the :file:`scripts/memfault` directory.
+
+To install the script's requirements, run the following command in its directory:
+
+.. code-block:: console
+
+   python3 -m pip install -r requirements.txt
+
+This script also requires an the nRF52 Series development kit to run the connectivity firmware that is programmed automatically when running the script.
+
+Using the script
+****************
+
+
+1. Connect your nRF52 Series development kit to your PC.
+#. Run the script commands in the :file:`scripts/memfault` directory.
+
+Your DK will be programmed with the connectivity firmware.
+To list the script arguments, run the following command:
+
+.. code-block:: console
+
+   python3 mds_ble_gateway.py -h
+
+The following help information describes the script arguments available:
+
+.. code-block:: console
+
+   Memfault BLE gateway
+
+   optional arguments:
+     -h, --help            show this help message and exit
```

Comment:
```suggestion
     -h, --help            Show this help message and exit
```

Reply:
It is printed by python argparser library so I don't want to change its default prompt